cloud/azure/templates/azure_saml_ses/variables.tf

```diff
@@ -3,6 +3,11 @@ variable "azure_resource_group" {
   description = "Name of the resource group where key vault is already created."
 }
 
+variable "civiform_time_zone_id" {
+  type        = string
+  description = "Time zone for Civiform server to use when displaying dates."
+}
```

Comment:
i'd probably add a default here just so it doesn't break the deploy our vars will overwrite the default but while this value gets defaulted in the conf file it also needs to be defaulted in the terraform (otherwise terraform will complain)

Reply:
Done!